pyproject.toml

```
@@ -9,15 +9,12 @@ build-backend = 'hatchling.build'
 name = 'pre-commit-terraform'
 classifiers = [
   'License :: OSI Approved :: MIT License',
-  'Programming Language :: Python :: 2',
-  'Programming Language :: Python :: 2.7',
   'Programming Language :: Python :: 3',
-  'Programming Language :: Python :: 3.6',
-  'Programming Language :: Python :: 3.7',
+  'Programming Language :: Python :: 3 :: Only',
   'Programming Language :: Python :: Implementation :: CPython',
   'Programming Language :: Python :: Implementation :: PyPy',
 ]
-description = 'Pre-commit hooks for terraform_docs_replace'
+description='Pre-commit hooks for terraform'
```

Terraform is a proper name. Just like Pre-commit. Either capitalize both or lowercase both please.
```suggestion
description='Pre-commit hooks for Terraform'
```

---

```suggestion
description = 'Pre-commit hooks for Terraform'
```